scripts/src/internal/services.ts

```diff
@@ -18,24 +18,33 @@ import { Application, AppOffer } from "../models/applications";
 
 const BLOCKCHAIN = "kin-prod";
 const RS512_APPS = ["test", "smpl"];
+const ACCOUNT_EXISTS_REASON = "account exists";
```

Comment:
If this is the blockchain error message I would rename this to: `ACCOUNT_EXISTS_BLOCKCHAIN_ERROR` because it's not obvious what this is and where it originates from.

Reply:
It's the message that the payment-service sends when the reason why account creation failed is that the account already exists. We currently send a single string as the reason. A better solution would be to send a code (strings can change). But adding this extra piece of info feels out of scope